src/components/NavBar.tsx

```diff
@@ -40,21 +39,21 @@ export default function SideBar(props: any) {
 
   return (
     <div>
-      <AppBar position="fixed" className={classes.appBar}>
-        <Toolbar style={{display: 'flex'}}>
+      <div style={{ display: 'fixed' }} className={classes.appBar}>
```

Comment:
why is this changed to div? Also don't use `style` here please, instead add any styling into `appBar` object.

Reply:
I think if we are going to use a `div` tag here in stead of the AppBar component we need to remove the margin from the top of the `Dashboard` component content otherwise everything gets offset for me. 

https://github.com/ethersphere/bee-status/blob/34d2dfda5a89fc0b103d6aabbd2c1af5e06215a6/src/layout/Dashboard.tsx#L16

![image](https://user-images.githubusercontent.com/40722304/111557251-b2b9d500-8762-11eb-92a8-1ced7ea34de4.png)